tests/yam/agama/boot_agama.pm

```diff
@@ -71,6 +80,7 @@ sub run {
             record_info('bootloader_zkvm');
             $self->bootloader_zkvm::run();
         }
+        set_agama_version;
```

Comment:
let's be consistent with the `()` in general is better to be explicit that is a function call to avoid surprises.